translib/transformer/xfmr_mclag.go

```diff
@@ -0,0 +1,266 @@
+////////////////////////////////////////////////////////////////////////////////
+//                                                                            //
+//  Copyright 2019 Dell, Inc.                                                 //
+//                                                                            //
+//  Licensed under the Apache License, Version 2.0 (the "License");           //
+//  you may not use this file except in compliance with the License.          //
+//  You may obtain a copy of the License at                                   //
+//                                                                            //
+//  http://www.apache.org/licenses/LICENSE-2.0                                //
+//                                                                            //
+//  Unless required by applicable law or agreed to in writing, software       //
+//  distributed under the License is distributed on an "AS IS" BASIS,         //
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  //
+//  See the License for the specific language governing permissions and       //
+//  limitations under the License.                                            //
+//                                                                            //
+////////////////////////////////////////////////////////////////////////////////
+
+package transformer
+
+import (
+	log "github.com/golang/glog"
+	"github.com/openconfig/ygot/ygot"
+	"strconv"
+	"strings"
+	"translib/db"
+	"translib/ocbinds"
+)
+
+func init() {
+	XlateFuncBind("YangToDb_mclag_domainid_fld_xfmr", YangToDb_mclag_domainid_fld_xfmr)
```

Comment:
Why these field-transformers like (YangToDb_mclag_domainid_fld_xfmr) is required, since openconfig-mclag-annot.yang is not part of this PR? Also couldn't see any other open PR for that OC-yang mclag extension. Also KLISH cmds in other PR (https://github.com/Azure/sonic-mgmt-framework/pull/59) uses sonic-yang for configuration